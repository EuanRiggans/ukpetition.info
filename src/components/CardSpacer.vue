<template>
    <div>
        <br>
        <v-spacer/>
        <br>
    </div>
</template>
<script>
    export default {
        name: 'CardSpacer'
    }
</script>
