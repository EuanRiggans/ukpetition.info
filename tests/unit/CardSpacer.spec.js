// test/CardSpacer.spec.js

// Libraries
import Vue from 'vue'
import Vuetify from 'vuetify'

// Components
import CardSpacer from '@/components/CardSpacer'

// Utilities
import {
    mount,
    createLocalVue
} from '@vue/test-utils'

const localVue = createLocalVue()
Vue.use(Vuetify)

describe('CardSpacer.vue', () => {
    let vuetify

    beforeEach(() => {
        vuetify = new Vuetify()
    })

    it('should have a custom title and match snapshot', () => {
        const wrapper = mount(CardSpacer, {
            localVue,
            vuetify,
        })

        // With jest we can create snapshot files of the HTML output
        expect(wrapper.html()).toMatchSnapshot()
    })
})
