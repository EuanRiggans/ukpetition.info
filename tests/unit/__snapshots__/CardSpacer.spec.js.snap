// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardSpacer.vue should have a custom title and match snapshot 1`] = `
<div><br>
  <div class="spacer"></div> <br>
</div>
`;
